src/server/bullet.test.js

```diff
@@ -0,0 +1,42 @@
+const Bullet = require('./bullet');
+const Constants = require('../shared/constants');
+
+describe('Bullet', () => {
+  const directionUp = 0;
+  const directionDown = Math.PI;
```

Comment:
nit: move these two constants up to the top-level scope of the file and rename them `DIRECTION_UP` and `DIRECTION_DOWN`